crates/ln-dlc-node/src/node/dlc_channel.rs

```diff
@@ -258,12 +244,11 @@ where
 }
 
 #[autometrics]
-pub(crate) async fn process_pending_dlc_actions(
+pub(crate) fn process_pending_dlc_actions(
```

Comment:
🤔 I'm not sure about making this function `sync`.

I'm tempted to keep it `await` in the hope of using `rayon` parallelism eventually, because we don't need to send the messages one-by-one, especially if they go to different nodes.

Reply:
I don't think we need an `async` context to use `rayon` 🤔 

> I'm not sure about making this function sync.

I just don't see how it benefits from being `async`.